models/comment.go

```diff
@@ -0,0 +1,20 @@
+package models
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation"
+	"time"
+)
+
+type Comment struct {
+	Id        string
```

Comment:
`Id` -> `ID`

Reply:
install this https://github.com/golangci/golangci-lint and configure on the project and run it. 
run it before every pull request